features/attached_commands.feature

```diff
@@ -515,8 +515,11 @@ Feature: Command behaviour when attached to an UA subscription
               {
                 "code": 200,
                 "response": {
-                  "activityToken": "test",
-                  "activityID": "ac-id",
+                  "activityInfo": {
+                    "activityToken": "test",
+                    "activityID": "ac-id",
+                    "activityPingInterval": "ping-test"
```

Comment:
We should make this the appropriate type integer
this is what we currently get from their endpoint
```yaml
'activityPingInterval': 3600
```